devops/env/dev/kustomization.yaml

```diff
@@ -61,7 +65,13 @@ images:
     newTag: "3.18.2"
   - name: image-registry.openshift-image-registry.svc:5000/6e2f55-tools/backup-container
     newTag: "latest"
-  - name: image-registry.openshift-image-registry.svc:5000/6e2f55-tools/reggie-web
-    newTag: "latest"
-  - name: image-registry.openshift-image-registry.svc:5000/6e2f55-tools/reggie-api
-    newTag: "latest"
+  #- name: image-registry.openshift-image-registry.svc:5000/6e2f55-tools/reggie-web
+  #  newTag: "latest"
+  #- name: image-registry.openshift-image-registry.svc:5000/6e2f55-tools/reggie-api
+  #  newTag: "latest"
+  - name: reggie-web
+    digest: sha256:e4c420c4d55e3c2f1f68d970d4c9e359ed0777bbc7abd707fde8a99fb6ed819f
```

Comment:
is this just the placeholder now? and it's get updated with `kustomize edit set image...` from the workflow

Reply:
Yes, the `name` field is the placeholder that is used for the base file's `image` field and the digest value is what is updated by `kustomize edit set image`.